### DISCUSSION TITLE
Precision X1 + FanControl

### DISCUSSION THREAD

**Discussion by phantum29** [Rem0o/FanControl.Releases#1333] [Q&A] [answered]
Looks like I need to close Precision X1 in order to utilize FanControl for the GPU.

I have a EVGA 3080ti FTW3 Ultra (converted to hybrid, OEM vbios).

This question is two fold. Firstly, when running FanControl detecting controllable fans, it recognizes the GPU but reports it cannot get a speed and cannot control. However when I show the fans (un-hide), I can control them, and rpm is reported. This is with X1 closed. With it opened, of course it takes over the fan curve. I've tried setting it to 0% on X1, but the FanControl curve seems to be off for what I had set. This is after finding start/stop % for the pump and fans. Is there a way to keep X1 open for undervolting / overclocking, while leaving FanControl to manage the fan speeds?

Secondly, is there something I should be doing differently to get FanControl to recognize that the fans are controllable and that it is reporting speed?

Thank you for the software, been loving it so far.

**Comment by Rem0o** [comment, marked as answer]
If you did the setup with X1 opened, the setup processed got overridden by X1 for the GPU, hence the no speed reported. That's one thing.

As for keeping x1 open, try and set the fan speed to "auto", that should be the default GPU behavior, and FanControl should be able to take over.
